LICENSE

```diff
@@ -1,6 +1,13 @@
-MIT License
+Dual License of MIT or LGPL
```

Comment:
It must be MIT *and* LGPL, as portions are covered under LGPL; downstream users cannot choose between MIT or LGPL at their prerogative (that would require permission from all other copyright holders).

Reply:
The old `nose` code is LGPL`. The new `pynose` code that wasn't in `nose` is MIT.

Reply:
Setting aside concerns of whether you can license as MIT changes that are most likely legally derivative works of LGPL code, that would still mean that the combination of your new code and the code from nose is covered under the conjunction of the LGPL and MIT licences. It would not be “dual‐licensed” in the conventional sense, where a consumer of the code can pick LGPL *or* MIT; instead, consumers would have to abide by the terms of both licences simultaneously. “Dual License of MIT or LGPL” is legally impossible in this case.